Sources/BraintreeUIKit/Components/BTUIKCardNumberFormField.m

```diff
@@ -50,18 +50,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
         [self setAccessoryViewHidden:YES animated:NO];
         
         self.validateButton = [UIButton new];
+
         [self.validateButton setTitle:BTUIKLocalizedString(NEXT_ACTION) forState:UIControlStateNormal];
-        
-        NSAttributedString *normalValidateButtonString = [[NSAttributedString alloc] initWithString:BTUIKLocalizedString(NEXT_ACTION)
-                                                                                         attributes:@{NSForegroundColorAttributeName:[BTUIKAppearance sharedInstance].tintColor,
-                                                                                                      NSFontAttributeName:[[BTUIKAppearance sharedInstance].boldFont fontWithSize:UIFont.labelFontSize]}];
-        [self.validateButton setAttributedTitle:normalValidateButtonString
-                                       forState:UIControlStateNormal];
-        NSAttributedString *disabledValidateButtonString = [[NSAttributedString alloc] initWithString:BTUIKLocalizedString(NEXT_ACTION)
-                                                                                           attributes:@{NSForegroundColorAttributeName:[BTUIKAppearance sharedInstance].disabledColor,
-                                                                                                        NSFontAttributeName:[[BTUIKAppearance sharedInstance].boldFont fontWithSize:UIFont.labelFontSize]}];
-        [self.validateButton setAttributedTitle:disabledValidateButtonString forState:UIControlStateDisabled];
+        [self.validateButton setTitleColor:[BTUIKAppearance sharedInstance].tintColor forState:UIControlStateNormal];
+        [self.validateButton setTitleColor:[BTUIKAppearance sharedInstance].disabledColor forState:UIControlStateDisabled];
 
+        self.validateButton.titleLabel.font = [BTUIKAppearance sharedInstance].staticHeadlineFont;
```

Comment:
Using an attributed title didn't seem necessary here, so we decided to simplify the code a bit.